test-apps/imjs-importer/src/ImportIMJS.ts

```diff
@@ -136,7 +136,7 @@ export class ImportIMJS {
             // console.log(g1);
             this.iModelDb.elements.insertElement(featureProps);
             const featureModel: SpatialModel = this.iModelDb.models.getModel(physicalModelId);
-            const featureModelExtents = featureModel.queryExtents();
+            const featureModelExtents = featureModel.queryExtents(); // eslint-disable-line deprecation/deprecation
```

Comment:
nit:

should we really be disabling the deprecation warning instead of just updating to the use the new api?
i know these are just test-apps, but im worried users will just blindly copy code

Reply:
I waffled a bit on deprecating `queryExtents`. For a backend-only app that creates a local snapshot or downloads a local briefcase, it's not horrible to query the extents on the main thread. I did not want to rewrite test apps' entire "main" routines to make them async so they can call the async extents query instead.

Reply:
Prefer adding a warning to the synchronous func instead of deprecating?

Reply:
@pmconne yeah, I think that probably puts us more in line with how node handles these things... plus I don't know that we really intend to ever delete this...

Reply:
Done.